docs/reference/redirects.asciidoc

```diff
@@ -1802,4 +1794,26 @@ See <<security-api-kibana-enrollment>>.
 See the <<sql-search-api-request-body,request body parameters>> for the
 <<sql-search-api,SQL search API>>.
 
+[role="exclude",id="restart-upgrade"]
+=== Full cluster restart upgrade
+
+When upgrading to {es} 8.0 and later, you must first upgrade to {prev-major-last}
+even if you opt to perform a full-cluster restart instead of a rolling upgrade.
+For more information about upgrading, see 
+{stack-ref}/upgrading-elastic-stack.html[Upgrading to Elastic {version}].
+
+role="exclude",id="rolling-upgrade"]
+=== Rolling upgrade
+
+When upgrading to {es} 8.0 and later, you must first upgrade to {prev-major-last}
+whether you opt to perform a rolling upgrade (upgrade one node at a time) or a full-cluster restart upgrade.
+For more information about upgrading, see 
+{stack-ref}/upgrading-elastic-stack.html[Upgrading to Elastic {version}]. 
+
+role="exclude",id="archived-settings"]
+=== Archived settings
 
+If you upgrade an {es} cluster that uses deprecated cluster or index settings
+that are not used in the target version, they are archived and ignored.
+For more information, see 
+{ref}/setup-upgrade.html#archived-settings[Archived settings]. 
```

Comment:
There is no archived settings section on this page.

Should the link go to this page:  https://www.elastic.co/guide/en/elasticsearch/reference/master/archived-settings.html